--- lalmetaio-4.0.4/Makefile.am.orig	2024-06-04 18:56:42.790285372 +0200
+++ lalmetaio-4.0.4/Makefile.am	2024-06-04 18:57:03.636839103 +0200
@@ -26,11 +26,6 @@ user_environment = \
 	trailingcolon MANPATH; \
 	set $(PACKAGE)_PREFIX $(prefix); \
 	set $(PACKAGE)_DATADIR $(pkgdatadir); \
-	prepend MANPATH $(mandir); \
-	prepend OCTAVE_PATH $(octexecdir); \
-	prepend PATH $(bindir); \
-	prepend PKG_CONFIG_PATH $(libdir)/pkgconfig; \
-	prepend PYTHONPATH $(pyexecdir) $(pythondir); \
 	$(END_OF_LIST)
 
 pkgconfigdir = $(libdir)/pkgconfig
